[2024-05-16T11:19:49.020+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-16T11:19:49.048+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_tables.import_table2 scheduled__2024-05-08T00:00:00+00:00 [queued]>
[2024-05-16T11:19:49.056+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_tables.import_table2 scheduled__2024-05-08T00:00:00+00:00 [queued]>
[2024-05-16T11:19:49.062+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-05-16T11:19:49.137+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): import_table2> on 2024-05-08 00:00:00+00:00
[2024-05-16T11:19:49.149+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_tables', 'import_table2', 'scheduled__2024-05-08T00:00:00+00:00', '--job-id', '398', '--raw', '--subdir', 'DAGS_FOLDER/dag_mysql.py', '--cfg-path', '/tmp/tmp1otzrtyc']
[2024-05-16T11:19:49.154+0000] {standard_task_runner.py:91} INFO - Job 398: Subtask import_table2
[2024-05-16T11:19:49.155+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=243) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-16T11:19:49.162+0000] {standard_task_runner.py:63} INFO - Started process 258 to run task
[2024-05-16T11:19:49.215+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_tables.import_table2 scheduled__2024-05-08T00:00:00+00:00 [running]> on host 7084a6d88d3b
[2024-05-16T11:19:49.347+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='bses' AIRFLOW_CTX_DAG_ID='dag_tables' AIRFLOW_CTX_TASK_ID='import_table2' AIRFLOW_CTX_EXECUTION_DATE='2024-05-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-08T00:00:00+00:00'
[2024-05-16T11:19:49.349+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-16T11:19:49.358+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-16T11:19:49.359+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n                mysql -h host.docker.internal -u root --password=12345678 test < /opt/***/dags/product_category.sql;\n                ']
[2024-05-16T11:19:49.367+0000] {subprocess.py:86} INFO - Output:
[2024-05-16T11:19:49.473+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-16T11:19:49.475+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-16T11:19:49.504+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_tables, task_id=import_table2, execution_date=20240508T000000, start_date=20240516T111949, end_date=20240516T111949
[2024-05-16T11:19:49.575+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-16T11:19:49.600+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-16T11:19:49.602+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-05-16T11:22:31.497+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-16T11:22:31.527+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_tables.import_table2 scheduled__2024-05-08T00:00:00+00:00 [queued]>
[2024-05-16T11:22:31.536+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_tables.import_table2 scheduled__2024-05-08T00:00:00+00:00 [queued]>
[2024-05-16T11:22:31.537+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-05-16T11:22:31.550+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): import_table2> on 2024-05-08 00:00:00+00:00
[2024-05-16T11:22:31.555+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=511) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-16T11:22:31.556+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_tables', 'import_table2', 'scheduled__2024-05-08T00:00:00+00:00', '--job-id', '450', '--raw', '--subdir', 'DAGS_FOLDER/dag_mysql.py', '--cfg-path', '/tmp/tmpp6hf3z73']
[2024-05-16T11:22:31.558+0000] {standard_task_runner.py:63} INFO - Started process 530 to run task
[2024-05-16T11:22:31.559+0000] {standard_task_runner.py:91} INFO - Job 450: Subtask import_table2
[2024-05-16T11:22:31.619+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_tables.import_table2 scheduled__2024-05-08T00:00:00+00:00 [running]> on host 7084a6d88d3b
[2024-05-16T11:22:31.709+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='bses' AIRFLOW_CTX_DAG_ID='dag_tables' AIRFLOW_CTX_TASK_ID='import_table2' AIRFLOW_CTX_EXECUTION_DATE='2024-05-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-08T00:00:00+00:00'
[2024-05-16T11:22:31.710+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-16T11:22:31.723+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-16T11:22:31.727+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n                mysql -h host.docker.internal -u root --password=12345678 test < /opt/***/dags/product_category.sql\n                ']
[2024-05-16T11:22:31.736+0000] {subprocess.py:86} INFO - Output:
[2024-05-16T11:22:31.887+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-16T11:22:31.893+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-16T11:22:31.924+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_tables, task_id=import_table2, execution_date=20240508T000000, start_date=20240516T112231, end_date=20240516T112231
[2024-05-16T11:22:31.970+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-16T11:22:32.000+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-16T11:22:32.017+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-05-16T11:24:46.781+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-16T11:24:46.800+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_tables.import_table2 scheduled__2024-05-08T00:00:00+00:00 [queued]>
[2024-05-16T11:24:46.805+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_tables.import_table2 scheduled__2024-05-08T00:00:00+00:00 [queued]>
[2024-05-16T11:24:46.807+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-05-16T11:24:46.831+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): import_table2> on 2024-05-08 00:00:00+00:00
[2024-05-16T11:24:46.838+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=745) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-16T11:24:46.839+0000] {standard_task_runner.py:63} INFO - Started process 758 to run task
[2024-05-16T11:24:46.838+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_tables', 'import_table2', 'scheduled__2024-05-08T00:00:00+00:00', '--job-id', '493', '--raw', '--subdir', 'DAGS_FOLDER/dag_mysql.py', '--cfg-path', '/tmp/tmpu6y90ju_']
[2024-05-16T11:24:46.843+0000] {standard_task_runner.py:91} INFO - Job 493: Subtask import_table2
[2024-05-16T11:24:46.893+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_tables.import_table2 scheduled__2024-05-08T00:00:00+00:00 [running]> on host 7084a6d88d3b
[2024-05-16T11:24:46.945+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='bses' AIRFLOW_CTX_DAG_ID='dag_tables' AIRFLOW_CTX_TASK_ID='import_table2' AIRFLOW_CTX_EXECUTION_DATE='2024-05-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-08T00:00:00+00:00'
[2024-05-16T11:24:46.948+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-16T11:24:46.955+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-16T11:24:46.957+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n                mysql -h host.docker.internal -u root --password=12345678 test < /opt/***/dags/product_category.sql\n                ']
[2024-05-16T11:24:46.964+0000] {subprocess.py:86} INFO - Output:
[2024-05-16T11:24:47.011+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-16T11:24:47.012+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-16T11:24:47.026+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_tables, task_id=import_table2, execution_date=20240508T000000, start_date=20240516T112446, end_date=20240516T112447
[2024-05-16T11:24:47.065+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-16T11:24:47.074+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-16T11:24:47.074+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-05-16T11:32:50.230+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-16T11:32:50.244+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_tables.import_table2 scheduled__2024-05-08T00:00:00+00:00 [queued]>
[2024-05-16T11:32:50.248+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_tables.import_table2 scheduled__2024-05-08T00:00:00+00:00 [queued]>
[2024-05-16T11:32:50.248+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-05-16T11:32:50.253+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): import_table2> on 2024-05-08 00:00:00+00:00
[2024-05-16T11:32:50.257+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=1142) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-16T11:32:50.258+0000] {standard_task_runner.py:63} INFO - Started process 1143 to run task
[2024-05-16T11:32:50.257+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_tables', 'import_table2', 'scheduled__2024-05-08T00:00:00+00:00', '--job-id', '542', '--raw', '--subdir', 'DAGS_FOLDER/dag_mysql.py', '--cfg-path', '/tmp/tmpotw8bfh5']
[2024-05-16T11:32:50.258+0000] {standard_task_runner.py:91} INFO - Job 542: Subtask import_table2
[2024-05-16T11:32:50.279+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_tables.import_table2 scheduled__2024-05-08T00:00:00+00:00 [running]> on host 7084a6d88d3b
[2024-05-16T11:32:50.319+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='bses' AIRFLOW_CTX_DAG_ID='dag_tables' AIRFLOW_CTX_TASK_ID='import_table2' AIRFLOW_CTX_EXECUTION_DATE='2024-05-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-08T00:00:00+00:00'
[2024-05-16T11:32:50.319+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-16T11:32:50.325+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-16T11:32:50.325+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n                mysql -h host.docker.internal -u root --password=12345678 test < /opt/***/dags/product_category.sql\n                ']
[2024-05-16T11:32:50.329+0000] {subprocess.py:86} INFO - Output:
[2024-05-16T11:32:50.384+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-16T11:32:50.385+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-16T11:32:50.398+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_tables, task_id=import_table2, execution_date=20240508T000000, start_date=20240516T113250, end_date=20240516T113250
[2024-05-16T11:32:50.447+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-16T11:32:50.462+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-16T11:32:50.463+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
